frontend/src/libs/helpers/get-time-from-creation/get-time-from-creation.helper.ts

```diff
@@ -0,0 +1,30 @@
+import { ConvertedTimeUnit, TimeKeyword } from './enums/enums.js';
+
+const getTimeFromCreation = (isoDate: string): string => {
```

Comment:
You've already added date-fns - just use `differenceIn{days/hours/minutes/seconds}` from it to return needed diff
It may looks like this
```
 const getDateDifferenceWithUnit = (date: string): { diff: number; unit: 'day' | 'hour' | 'minute' | 'second' } => {
    const currentDate = new Date();

    const diffInDays = differenceInDays(currentDate, date);
    if (diffInDays) {
      return { diff: diffInDays, unit: 'day' };
    }

    const diffInHours = differenceInHours(currentDate, date);
    if (diffInHours) {
      return { diff: diffInHours, unit: 'hour' };
    }

    const diffInMinutes = differenceInMinutes(currentDate, date);
    if (diffInMinutes) {
      return { diff: diffInMinutes, unit: 'minute' };
    }

    const diffInSeconds = differenceInSeconds(currentDate, date);
    return { diff: diffInSeconds, unit: 'second' };
 }
```